packages/compile-common/src/profiler/profiler.ts

```diff
@@ -10,8 +10,8 @@ import { requiredSources, RequiredSourcesOptions } from "./requiredSources";
 import { convertToAbsolutePaths } from "./convertToAbsolutePaths";
 
 export interface ProfilerConfig {
-  parseImports: RequiredSourcesOptions["parseImports"];
-  shouldIncludePath: RequiredSourcesOptions["shouldIncludePath"];
+  parseImports?: RequiredSourcesOptions["parseImports"];
+  shouldIncludePath?: RequiredSourcesOptions["shouldIncludePath"];
```

Comment:
Here's `compile-vyper` creating an instance of profiler Profiler without them. For `compile-ligo` and `compile-michelson` I need to create it with `shouldIncludePath` but without `parseImports`.